package.json

```diff
@@ -29,9 +29,10 @@
     "start": "nf start",
     "start:dev": "NODE_ENV=development COMBINED=true babel-watch -L server",
     "ngrok": "ngrok http 8082 --region eu --subdomain sql",
-    "test": "npm run test:lint && npm run test:flow && npm run test:units",
+    "test": "npm run test:lint && npm run test:flow && npm run test:unit && npm run test:integration",
     "test:lint": "eslint server",
-    "test:units": "NODE_ENV=test mocha --require babel-register -R spec ./tests",
+    "test:integration": "NODE_ENV=test mocha --require babel-register -R spec ./tests/integration",
+    "test:unit": "NODE_ENV=test mocha --require babel-register -R spec ./tests/unit",
```

Comment:
@mkawalek thank you for making the structure match the convention - watch out it should be `/test` instead of `/tests`